scss/components/_tooltips.scss

```diff
@@ -0,0 +1,88 @@
+//tooltip position
+$ico-size: 23px; //get from _xsmall.scss
+$ico-size-small: 25px * $xsmall-multiple; //calc get from _xsmall.scss
+$items-wrapper-margin-top: 10px;
+$tooltip-offset: $ico-size + $items-wrapper-margin-top + ($scrubber-bar-height/2 + $scrubber-bar-padding-height/2);
+$tooltip-offset-small: $ico-size-small + $items-wrapper-margin-top + ($scrubber-bar-height-small/2 + $scrubber-bar-padding-height-small/2);
+
+//tooltip
+$tooltip-opacity: 0.75;
+$tooltip-color: black;
+$tooltip-animation-duration: 0.5s;
+$tooltip-pointer-size: 5px;
+
+.oo-tooltip::before, .oo-tooltip::after{
+  position: absolute;
+  z-index: $zindex-popover;
```

Comment:
I would recommend defining a `zindex-tooltip` var with the same value as `zindex-popover` so that the two are not tied together. Did this work fine with ads btw?